docs/pages/application-access/okta.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Okta integration
+description: Guides for using Teleport Okta integration.
+layout: tocless-doc
+---
+
+These guides explain how to use the Okta service to integrate teleport with
```

Comment:
integrate teleport with okta in order to do what? The fact that there's an integration is obvious from the title. Is there something more descriptive we can say here?